infrastructure/network/netadapter/server/grpcserver/protowire/rpc.md

```diff
@@ -1817,6 +1819,33 @@ See NotifyNewBlockTemplateRequestMessage
 
 
 
+
+<a name="protowire.GetCoinSupplyRequestMessage"></a>
+
+### GetCoinSupplyRequestMessage
+
+
+
+
+
+
+
+<a name="protowire.GetCoinSupplyResponseMessage"></a>
+
+### GetCoinSupplyResponseMessage
+
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| totalSompi | [uint64](#uint64) |  | note: this is a hard coded totalSupply, actual totalSupply is expected to deviate by upto -5%, but cannot be measured exactly. |
```

Comment:
Why can't it be measured exactly?

Reply:
you even mention this on kaspanet.org i.e. a few hundred million kas will always be +/- . this is due to daa window and deflationary daa scores, i think.